Bitwarden/Application/Support/Bitwarden.entitlements

```diff
@@ -26,6 +26,7 @@
 	<key>com.apple.security.application-groups</key>
 	<array>
 		<string>group.$(BASE_BUNDLE_ID)</string>
+		<string>group.$(SHARED_GROUP_ORGANIZATION_IDENTIFIER).bitwarden-authenticator</string>
```

Comment:
⚠️ The group depends on whether the app is being built on PRD or Beta variant so I would extract the suffix `bitwarden-authenticator` to the `xcconfig` as well and update it accordingly in the `select_variant.sh` script.

Reply:
Sounds good. I'll add support for that in a commit. 👍

Reply:
@fedemkr Can you take a look at my last commit? I think this has everything we need but let me know if you want any changes. I'll make the same change to the Authenticator PR I have open as well.